cliqz-logger.js

```diff
@@ -0,0 +1,419 @@
+// First two arguments represent absolute path to node executable itself and
+// to cliqz-logger.js file.
+// So we are interested in arguments from the 3rd one;
+const inputArgs = Array.prototype.slice.call(process.argv, 2);
+
+const writeToConsole = function() {
```

Comment:
What about:
```suggestion
const writeToConsole = function(...messages) {
```

Reply:
@remusao In browser project we do not use latest ES6 features for now.
So I will stick to ES5 in this case.

Reply:
Well, I can understand why not in browser code but this is a stand-alone script. Anyways, you guys decide :)

Reply:
Starting from node.js v8.11.3 is should work fine.